### PR TITLE
Done 1, 2, 3, 4

### PR DESCRIPTION
**Pull request by Romchik38**

### CODE REVIEW

File: Exercises/3-unique.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 'use strict';
 
-// Create and return a new array without duplicate elements
-// Don't modify initial array
-
-const unique = array => [];
+const unique = array => {
+  const uniq = [];
+  for (const item of array) {
+    const index = uniq.indexOf(item);
```
Comment (tshemsedinov) [inline Exercises/3-unique.js]: Use `.includes(item)` method

File: Exercises/4-difference.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-// Find difference of two arrays
-// elements from array1 that are not includes in array2
-
-const difference = (array1, array2) => [];
+const difference = (array1, array2) => array1.filter(elem => {
```
Comment (tshemsedinov) [inline Exercises/4-difference.js]: Try to reduce complexity:
- Don't create intermediate array `duplicate`
- You can implement it using just resulting array and for..of loop

**Review by tshemsedinov — CHANGES_REQUESTED** [review]

File: Exercises/3-unique.js (hunk 1 above)
Comment (Romchik38) [reply Exercises/3-unique.js]: done

File: Exercises/4-difference.js (hunk 2 above)
Comment (Romchik38) [reply Exercises/4-difference.js]: done

**Review by tshemsedinov — APPROVED** [review]
:+1:

**Pull request closed without merging**